### PR TITLE
Fixes #21 - Pathway views

### LINKED ISSUE #21: Build Pathways views
In the original mockups, there are views for when the user clicks on Pathways in the nav.

Pathways: https://redpen.io/qs5ea730d5d8195f89

They include each of the Pathways with a link to view all sessions in that Pathway.

mozfest 2016 mobile scheduler-04

mozfest 2016 mobile scheduler-06

### PR DESCRIPTION
**Pull request by mmmavis**

### CODE REVIEW

**Comment by mmmavis** [comment]
@ryanpitts wondering if you have time to look over this PR on Monday? :smile:

**Comment by ryanpitts** [comment]
Yes, absolutely. I have a call this morning, but will look at this right after it's done.

File: js/schedule.js [hunk 1]
```diff
@@ -535,14 +540,59 @@ function Schedule(options) {
         schedule.getFilteredSessions("space", space_slug);
     }
 
+    // display the list of Spaces and their descriptions
```
Comment (ryanpitts) [inline js/schedule.js]: not a big deal, but comment should probably say "pathways" instead of "spaces"

File: js/schedule.js [hunk 2]
```diff
@@ -535,14 +540,59 @@ function Schedule(options) {
...
+        }
+    }
+
+    schedule.appendPathwayListItems = function() {
+        schedule.clearHighlightedPage();
+        schedule.$pageLinks.find('#pathways-page-link').addClass('active');
+        schedule.$container.html("");
+        schedule.addCaptionOverline("<h3><span>Pathways</span></h3>");
+
+        _.each(_.sortBy(_.keys(schedule.pathwayMap)), function(v, k) {
```
Comment (ryanpitts) [inline js/schedule.js]: <3 underscore :)

**Comment by ryanpitts** [comment]
Looks great, I say merge away! :+1:

File: js/schedule.js (hunk 1 above)
Comment (mmmavis) [inline js/schedule.js]: will have this edited :+1:

**Comment by mmmavis** [comment]
yay merging this now :tada:

**Pull request merged**